Update 2409Integration-Int-Ext.py

mefit0 opened the pull request:
* i added the internal retina/ internal environment (goal) and the plots. The internal retina moves like the external one, with the same coordinates on the "field of view". Note that imshow for the plots works this way: if there are 0-1 in the matrix it will use blue and red color; if there are more numbers in the matrix it adds other colors like yellow, light blue etc; so when the retina is on the 3 figures you will see all the "shades" of color, that changes if the retina is only on one of the figures (the image will be again just in red and blue). Note that it's NOT a problem of matrix but just a feature of imshow() that we don't need to fix now, it would be a waste of time!

close #2

mefit0 committed:
Update 2409Integration-Int-Ext.py

* i added the internal retina/ internal environment (goal) and the plots. The internal retina moves like the external one, with the same coordinates on the "field of view". Note that imshow for the plots works this way: if there are 0-1 in the matrix it will use blue and red color; if there are more numbers in the matrix it adds other colors like yellow, light blue etc; so when the retina is on the 3 figures you will see all the "shades" of color, that changes if the retina is only on one of the figures (the image will be again just in red and blue). Note that it's NOT a problem of matrix but just a feature of imshow() that we don't need to fix now, it would be a waste of time!

close #2

wlord01 reviewed (approved):
Nice! Well done Ilaria 👍 :) Let's merge this to the master, if @NeuroDoctor doesn't have any comments?

A few points:
* This is written in Python 2, so I had to add a parenthesis to print calls since I use Python 3. We should use the same version. Should we go with 2 or 3?
* I think we should organize the code in separate scripts to make it easier to understand and easier to test new things, according to the links I sent you on WhatsApp. Ilaria and I discussed this already and I think we agreed on this. This goes for the classes and existing functions, but I also think that we can make functions out of more of the code as it is now. 

Thanks for this commit :)

wlord01 committed:
Put polygon objects in package polygonobjects

In order to divide the code into different files for better overview and
easier testing/running I created the package 'polygonobjects' and moved
the polygon object classes in there.

wlord01 commented on 2409Integration-Int-Ext.py:
```diff
@@ -42,115 +42,9 @@
-        return pixel_list
+from polygonobjects import Polygon, Circle, Hand, Retina
+
+plt.ion()
```
Mind that I added plt.ion() for the animations, since I work in IPython.

wlord01 commented on 2409Integration-Int-Ext.py:
```diff
@@ -504,7 +398,7 @@ def pixelLoop(grid,tri,squ,cir,retiPixVec):
                 #for now saccade num = hand movements num EXCEPT for the last
                 #movement of "release" of the object
-                print "\nMOVE, at step: ", numTri
+                print("\nMOVE, at step: ", numTri)
```
Mind that I added parentheses here to the print call, since I work in Python 3.
